diff --git a/Application/loopback/loopback.c b/Application/loopback/loopback.c
index 5f3dff6..a4b6dc2 100644
--- a/Application/loopback/loopback.c
+++ b/Application/loopback/loopback.c
@@ -1,7 +1,8 @@
 #include <stdio.h>
 #include "loopback.h"
-#include "socket.h"
-#include "wizchip_conf.h"
+
+#include "io6Library/Ethernet/socket.h"
+#include "io6Library/Ethernet/wizchip_conf.h"
 #include "stdlib.h"
 
 #if LOOPBACK_MODE == LOOPBACK_MAIN_NOBLCOK
diff --git a/Internet/DHCP4/dhcpv4.c b/Internet/DHCP4/dhcpv4.c
index 980bb11..4b837d8 100644
--- a/Internet/DHCP4/dhcpv4.c
+++ b/Internet/DHCP4/dhcpv4.c
@@ -50,7 +50,7 @@
 //
 //*****************************************************************************
 
-#include "socket.h"
+#include "io6Library/Ethernet/socket.h"
 #include "dhcpv4.h"
 
 /* If you want to display debug & processing message, Define _DHCPV4_DEBUG_ in dhcp.h */
diff --git a/Internet/DNS/dns.c b/Internet/DNS/dns.c
index c597fad..0904e14 100644
--- a/Internet/DNS/dns.c
+++ b/Internet/DNS/dns.c
@@ -52,7 +52,7 @@
 #include <string.h>
 #include <stdlib.h>
 
-#include "socket.h"
+#include "ioLibrary_Driver/Ethernet/socket.h"
 #include "dns.h"
 
 #ifdef _DNS_DEBUG_
diff --git a/Internet/DNS/dns.h b/Internet/DNS/dns.h
index 279b9e3..d12db6c 100644
--- a/Internet/DNS/dns.h
+++ b/Internet/DNS/dns.h
@@ -68,7 +68,7 @@ extern "C" {
  * @todo SHOULD BE defined it equal as or greater than your Domain name lenght + null character(1)
  * @note SHOULD BE careful to stack overflow because it is allocated 1.5 times as MAX_DOMAIN_NAME in stack.
  */
-#define  MAX_DOMAIN_NAME   16       // for example "www.google.com"
+#define  MAX_DOMAIN_NAME   128
 
 #define	MAX_DNS_RETRY     2        ///< Requery Count
 #define	DNS_WAIT_TIME     3        ///< Wait response time. unit 1s.
